openrtb_ext/imp_iqzone.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ImpExtFoo struct {
```

Comment:
Even if it's not being used, please rename to something less generic, `ImpExtIqzone` maybe

Reply:
This just looks like a copy/paste error from the docs. Easy fix. Thank you @IQZoneAdx for faithfully following our new bid adapter guide. :)